online-dashboard-api/app/Http/Requests/UpdateJobRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Why to have same data in two request : CreateJobRequest and UpdateJobRequest
Use single Request as JobRequest, that would be fine,

Reply:
done